supports/data-sync-performance-test/log_ssm.py

```diff
@@ -0,0 +1,28 @@
+import re
+
+pstart = re.compile(r'\d\d:\d\d:\d\d.*Start rule\d')
+pend = re.compile(r'\d\d:\d\d:\d\d.*\n.*\d\d:\d\d:\d\d.*Stop rule\d')
+
+action = [[],[],[]]
+hack = 3
+a = ""
+with open("/root/smart-data-1.3.2/logs/smartserver.log",'r') as f:
```

Comment:
`/root/smart-data-1.3.2`  can be replaced with `$SMART_HOME=`. We can configure and export this parameter before running.